src/test/java/net/snowflake/ingest/connection/UserAgentTest.java

```diff
@@ -5,10 +5,12 @@
 import java.util.Properties;
 import net.snowflake.ingest.SimpleIngestManager;
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class UserAgentTest {
   @Test
+  @Ignore
```

Comment:
Why aren't we changing also `RequestBuilder.DEFAULT_VERSION`?

Reply:
Ah, we want the version in user agent without `-unshaded`, right?

Reply:
yup, at least that's what we did for the past few versions and we don't have an issue so far